spec/pluto/operation/vertical_blur_spec.cr

```diff
@@ -7,21 +7,28 @@ describe Pluto::Operation::VerticalBlur do
       blurred_image = image.vertical_blur(10)
 
       expect_digest image, "1a4d4e43e17f3245cefe5dd2c002fb85de079ae8"
-      expect_digest blurred_image, "45b4a49232674ed4e33c06e09c19ba22e0baa3c3"
+      expect_digest blurred_image, "38d71a5c13f46afdca6b13ecbdeb97327cd46dd7"
     end
 
     it "works with ImageRGBA" do
       image = rgba_sample
       blurred_image = image.vertical_blur(10)
 
       expect_digest image, "d7fa6faf6eec5350f8de8b41f478bf7e8d217fa9"
-      expect_digest blurred_image, "d50bbaba05b5a42ccd1b3c8e14092b10075b07e9"
+      expect_digest blurred_image, "d7116d6cea0a14e23cc3a23dbc86ad8bf1fecf2f"
     end
 
     it "doesn't cause arithmetic overload" do
       with_sample("problem_images/28_arithmetic_overflow_in_blur.jpg") do |io|
         image = Pluto::ImageRGBA.from_jpeg(io)
-        expect_digest image.vertical_blur(10), "64039b39a57fc260bfd433d3fc298a8ac5366938"
+        expect_digest image.vertical_blur(10), "fd483aa927fb2ca8fdeb714e82dbbd3bf173daba"
+      end
+    end
+
+    it "doesn't cause arithmetic overload" do
+      with_sample("problem_images/46_arithmetic_overflow_in_blur_again.jpg") do |io|
+        image = Pluto::ImageRGBA.from_jpeg(io)
+        expect_digest image.vertical_blur(5), "89039df3a08bef9ba379263cedcfdc27cc8c891d"
```

Comment:
Mistake on my part, the `vertical_blur` method doesn't throw an arithmetic overflow for this example image. Kept the spec because it didn't hurt anything.

Reply:
Ideally, I think specs should only test actual issues. So that if we ever come back to the question of why something was done this way and not another way, we have specs that we can refer to.

I would remove this if it does not actually cause arithmetic overflow. What do you think?

Reply:
Probably the correct choice and I'm just being lazy :grin: